src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -68,10 +64,7 @@
     <Compile Remove="coreclr\Math\Functions\Double\Asinh.cs" />
     <Compile Remove="coreclr\Math\Functions\Double\Atanh.cs" />
     <Compile Remove="coreclr\Math\Functions\Double\Cbrt.cs" />
-    <Compile Remove="coreclr\Math\Functions\Single\Acosh.cs" />
-    <Compile Remove="coreclr\Math\Functions\Single\Asinh.cs" />
-    <Compile Remove="coreclr\Math\Functions\Single\Atanh.cs" />
-    <Compile Remove="coreclr\Math\Functions\Single\Cbrt.cs" />
+    <Compile Remove="coreclr\Math\Functions\Single\**" />
```

Comment:
Why not `**\*.cs` at the end? If someone adds a .txt, .md, or .resx file to the folder, it will cause compiler errors.

Reply:
Good point, fixed

Reply:
Oh - at first look I missed these were `Remove` elements, so I guess it didn't matter much (since the other file types shouldn't have been in `Compile` to start with). But this is clearer IMO - thanks!